docs/loading_trained_model_for_inference.md

```diff
@@ -0,0 +1,49 @@
+In the previous tutorials, we have created, trained and saved a model using existing labeled data. 
+Now let's see how we can load and use that model to generate predictions on the new, previously unseen data. 
+
+While loading the model and calling `predict` method on it is going to be quite straightforward, there are a few 
+important steps that you need to do before that. 
+
+First, you need to make sure the image that you'll be using is of the same size. We've trained our model on monochrome 
+images 28 x 28 pixels. So you may need to crop, or resize you images, and convert them to grayscale in order to be able to 
+use with this model. As a developer, you're responsible for making sure, your input matches the model's requirements. 
+
+Here's an image of a bag I'll be using in this example, after the preprocessing:
+
+![](images/test-image-bag.png)
+
+The model expects as an input a 28 x 28 Tensor (Tensor == multi-dimensional array). We can read in the image as 
+a FloatArray, and define a function that will help us reshape a regular FloatArray into a Tensor with appropriate 
+dimensions:
+
+```kotlin
+val floatArray = ImageConverter.toRawFloatArray(File(PATH_TO_TEST_IMAGE))
+
+private fun reshape(floats: FloatArray): Tensor<*> {
```

Comment:
hmmm, seems like only one place where Tensors from low-level API are accessible, need to refactor and update later

I suppose that reshape function should return FloatArray (reshaped variable) instead of Tensor<*>

Reply:
Another thought that I had here - should the reshaping happen within loaded model? Or maybe it should be part of the preprocessing before the data gets fed into the model?